packages/website/src/components/RecruitmentModal/RecruitmentModal.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import { IconButton } from '@coderscamp/ui/components/IconButton';
```

Comment:
❓ Chakra has its own button for closing modals called `AlertDialogCloseButton`. Cannot we use it here?